src/createServer.js

```diff
@@ -1,3 +1,38 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { isSearchParamsValid } = require('./errorhandler');
+
+const createServer = http.createServer((req, res) => {
```

Comment:
createServer should be a function

```suggestion
const createServer = () => http.createServer((req, res) => {
```